fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraTransactions.java

```diff
@@ -49,9 +49,6 @@
 
```

Comment:
FedoraTransactions needs to be re-written to use FedoraTransactionManager instead of BatchService